tests/Tests/100__deploy/100__installation/107__alert_manager.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation     Post install test cases that verify OCP Alert Manager resources and objects
+Library           OpenShiftLibrary
+Resource          ../../../Resources/ODS.robot
+
+*** Test Cases ***
+Verify Alert Manager Is Shipped And Enabled Within ODS
```

Comment:
is there anything in the alertmanager configmap which might be worth to check here?

Reply:
In my opinion, alertmanager's configuration should have a test of its own, that could be added in another PR, does this make sense to you?

Reply:
I agree with Pablo that this test suite can be enhanced in the future